src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/logic/action/ActionDelegate.groovy

```diff
@@ -703,6 +703,15 @@ class ActionDelegate {
          },
          validations: { cl ->
              changeFieldValidations(field, cl, targetCase, targetTask)
+         },
+         componentProperties: { cl ->
+             def properties = cl()
+             if (properties == null || (properties instanceof Closure && properties() == UNCHANGED_VALUE))
```

Comment:
Always wrap one line body of if into braces

Reply:
done